meetings/2024-06-12.md

```diff
@@ -28,11 +35,123 @@
+* eP: Can we do a doodle to find a spot? Share the result with organizer to get maybe a second option.
+
+ACTION: PAC to create a doodle.
+  → https://evento.renater.fr/survey/solid-cg-meeting-at-tpac-2025-oat4znsh
+
+* PAC: do we put all slots, or filter first?
+* eP: prefer to add all and filter later
+* PAC: maybe give a deadline.
+* VB: (following minutes but cannot join): Our options are limited to what's available. Event will be added automatically to our calendar. Our first option of Monday was not available. Tuesday was not good for some people. Feel free to add all slots and we can ask what's possible but deadline was June 10th to submit, so our choices might no longer be available at this point. Let me know and I'll check. 
+* eP: sharing with the group what VB is writing.
+* PAC: we're past the deadline anyway
+* eP: 48 hour should suffice. we cand send list mail and @room mention on matrix (any chair can do it)
+* HZ: agreed 
+* VB: Perhaps focus on Thu and Fri, since we already know Mon and Tue don't work. 
+
+### initial WebID Document considerations
+
+URL: https://github.com/solid/security-considerations/pull/9
+
+* HZ: Jacopo is no longer a chair of WebID CG, not sure how it impacts Solid
+* eP: STM a week ago, JZ, VB, MDJ present. What's the criteria of accepting a proposal? For me, is to not conflict with the Solid protocol. Nobody showed me any problem with conformance. Discussing counter measures for attacks. My impression is that sometimes we put too much in a PR. If someone things something more needs to be added, they can create a new PR. IMHO this PR is ready to merge. If later we decide we can have more normative requirements, it can be changed.
+* JZ: I objec to this PR. We have nothing in place to describe what a webid document does. I agree that we need to take into account security and keep webid docs outside solid servers is a valid option. Recommending that without anything else in place is only going to create a mess. I think it's premature to now start talking about where it leaves without first describing what a webid doc is. There is no consistency.
+* eP: I am aware of webid profile draft. Not ignoring that this work exist. Webid profile spec is not completely useless if webid is stored outside Solid. I have this diagram (screen shared). Solid protocol has a normative reference to the webid spec. I think we agreed last week that we need to agree about discovery, make parts public, some protected, but we're not there yet. I am having my WebID ouside the storage. It's a well established practice.
+* JZ: I am not trying to prevent people from having the document outside storage. I tried for 6 months to add a predicate for allowing a webid doc to be stored outside Solid. I think the content of webid should be established first. Discovery is important. I don't want to discuss it later. That extends the period of ambiguity for developers.
```

Comment:
```suggestion
* JZ: I object to this PR. We have nothing in place to describe what a WebID document does. I agree that we need to take security into account, and keeping WebID docs outside Solid servers is a valid option. Recommending that without anything else in place is only going to create a mess. I think it's premature to start talking about where it leaves without first describing what a WebID doc is. There is no consistency.
* eP: I am aware of WebID Profile draft. Not ignoring that this work exists. WebID Profile spec is not completely useless if WebID is stored outside Solid. I have this diagram (screen shared). Solid Protocol has a normative reference to the WebID spec. I think we agreed last week that we need to agree about discovery, make parts public, some protected, but we're not there yet. I have my WebID Document outside the Solid storage. It's a well established practice.
* JZ: I am not trying to prevent people from having their WebID Document outside Solid storage. I tried for 6 months to add a predicate for allowing a WebID Document to be stored outside Solid. I think the content of WebID Document should be established first. Discovery is important. I don't want to discuss it later. That extends the period of ambiguity for developers.
```